frappe_pwa/www/pwa.js

```diff
@@ -4,6 +4,19 @@
  * Licensed under AGPL v3 (https://github.com/Monogramm/frappe_pwa/blob/master/LICENSE)
  */
 
+
+function showPrompt(buttonText, messageText, f) {
+    const $btn = $(`<button class="pwa-next-action"><span>${__(buttonText)}</span></button>`);
+    const next_action_container = $(`<div class="pwa-next-action-container"></div>`);
+    $btn.click(() => f());
+    next_action_container.append($btn);
+    show_pwa_alert({
+        message: __(messageText),
+        body: next_action_container,
+        indicator: 'green',
+    })
+}
```

Comment:
Group your functions together instead of splitting them one at start one at end of file